test/test.gyp

```diff
@@ -10,85 +10,92 @@
         'run-tests.c',
         'runner.c',
         'runner.h',
-        'test-get-loadavg.c',
         'task.h',
         'test-active.c',
-        'test-async.c',
         'test-async-null-cb.c',
-        'test-callback-stack.c',
+        'test-async.c',
+        'test-barrier.c',
         'test-callback-order.c',
+        'test-callback-stack.c',
         'test-close-fd.c',
         'test-close-order.c',
+        'test-condvar.c',
         'test-connect-unspecified.c',
         'test-connection-fail.c',
         'test-cwd-and-chdir.c',
         'test-default-loop-close.c',
         'test-delayed-accept.c',
+        'test-dlerror.c',
         'test-eintr-handling.c',
-        'test-error.c',
         'test-embed.c',
         'test-emfile.c',
         'test-env-vars.c',
+        'test-error.c',
         'test-fail-always.c',
         'test-fork.c',
-        'test-fs.c',
-        'test-fs-readdir.c',
         'test-fs-copyfile.c',
         'test-fs-event.c',
         'test-fs-fd-hash.c',
         'test-fs-open-flags.c',
         'test-fs-poll.c',
-        'test-getters-setters.c',
+        'test-fs-readdir.c',
+        'test-fs.c',
         'test-get-currentexe.c',
+        'test-get-loadavg.c',
         'test-get-memory.c',
         'test-get-passwd.c',
         'test-getaddrinfo.c',
         'test-gethostname.c',
         'test-getnameinfo.c',
         'test-getsockname.c',
+        'test-getters-setters.c',
         'test-gettimeofday.c',
         'test-handle-fileno.c',
         'test-homedir.c',
         'test-hrtime.c',
         'test-idle.c',
         'test-idna.c',
+        'test-ip4-addr.c',
+        'test-ip6-addr.c',
         'test-ip6-addr.c',
         'test-ipc-heavy-traffic-deadlock-bug.c',
         'test-ipc-send-recv.c',
         'test-ipc.c',
         'test-list.h',
-        'test-loop-handles.c',
         'test-loop-alive.c',
         'test-loop-close.c',
+        'test-loop-configure.c',
+        'test-loop-handles.c',
         'test-loop-stop.c',
         'test-loop-time.c',
-        'test-loop-configure.c',
-        'test-walk-handles.c',
-        'test-watcher-cross-stop.c',
         'test-multiple-listen.c',
+        'test-mutexes.c',
+        'test-not-readable-nor-writable-on-read-error.c',
+        'test-not-readable-on-eof.c'
```

Comment:
An interesting factoid about gyp files: if you omit the comma, it concatenates the strings. ^^

(The build failures are caused by the missing comma here.)